packages/website-router/titles-tags/config.ts

```diff
@@ -0,0 +1,8 @@
+import { TitleConfig } from './title-seo';
+
+export const MANAGE_TITLE: TitleConfig[] = [
+  {
+    titleSEO: 'GraphQL TypeScript',
+    URL: 'https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-operations',
```

Comment:
I think it might not be good to depend on the whole full URL. Why not just `/plugins/typescript/typescript-operations`?